src/test/java/entities/InvitationTest.java

```diff
@@ -0,0 +1,70 @@
+package entities;
+import entities.InvitationEntities.Invitation;
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**Test couldn't be run as some entities are not yet implemented*/
+//public class InvitationTest {
+//    String sentTime;
+//
+//    User sender;
+//    User receiver;
+//    Task task;
+//
```

Comment:
The tests are commented out because your code assumes that User is not implemented, right?